StreetDrop/StreetDrop/Data/DataMapping/PoiResponseDTO.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  PoiResponseDTO.swift
+//  StreetDrop
+//
+//  Created by 맹선아 on 2023/05/13.
+//
+
+import Foundation
+
+struct PoiResponseDTO: Decodable {
+    let allPOI: [Poi]
```

Comment:
뒤에 -s 붙이는걸로 통일하면 하기처럼 될 것 같습니다
```suggestion
    let pois: [Poi]
```

Reply:
아 놓친부분 잡아주셔서 감사합니다
수정완료했습니다.. :) 
[6322245](https://github.com/depromeet/street-drop-iOS/pull/20/commits/632224541b193a0ac1d5f630fb8dfbb272201346)